_faqs/faq-12.md

```diff
@@ -1,8 +1,110 @@
+The RIOT community can only start working on drivers once the MCUs are generally available and more exotic features that are only relevant on niche use cases may never be supported in favor of a cleaner and more portable API in RIOT.
+
+From a government point of view RIOT is developed by a self-governed community, while the vendor SDKs are under tight control of the vendor.
+The RIOT community has so for maintained many popular boards well after the vendors declared them obsolete, while vendor SDKs often phase out support for older MCUs sooner.
+
+Vendor specific SDKs typically do not have any copyleft clauses.
+
+## Arduino
+
+Both RIOT and Arduino have a strong focus on a gentle learning curve and a low entry barrier.
+Arduino however goes a step further and sacrifices advanced features such as native multi-threading, a full-fledged and mature network stack, or VFS subsystem for simplicity.
+This combined with a trimmed down IDE results in a lower entry barrier for Arduino compared to RIOT.
+
+RIOT still allows new users to quickly get productive, but without limiting developers to specific design patterns, specific software architectures, and a strong tie to a specific IDE.
+As a nice treat for users that started with Arduino but ran in limitations, our [Arduino compatibility layer][riot-arduino] allows re-using sketches and libraries from the Arduino world.
+
+Arduino has a vast ecosystem of libraries developed independently by 3rd parties.
+RIOT set of modules and packages is not that vast, but RIOT's modules are seamlessly integrated, have a consistent APIs and integrate well with each others.
+This results in easier reuse of code and higher productivity without having to worry about interoperability issues of components and "impedance matching" between misaligned APIs.
+
+The Arduino software is released under a copyleft license and hardware support can be extended greatly beyond the official Arduino boards by using 3rd party Arduino cores.
+The development of the ecosystem is sponsored and controlled by a single company that owns the registered trademark of Arduino, while RIOT governed by an independent community.
+
+## Zephyr
+
+Zephyr has a larger and more complex code base and a very sophistication configuration system and unique tooling and is backed by a larger community than RIOT.
+This results in a larger feature set at the expense of a steeper learning curve.
+
+RIOT leaner code base makes the code easier to follow and results in a lower memory footprint.
+In addition RIOT has been ported to 8-bit and 16-bit platforms, allowing its use on MCUs that are too resource constraint to run Zephyr.
+
+Zephyr's larger code base and development community does give Zephyr an edge when it comes to the number of features supported.
+We do believe that RIOT has an edge over zephyr when a gentle learning curve, memory consumption, or quick prototyping cycles are priorities.
```

Comment:
line 68: Zephyr instead of zephyr

Reply:
Thx, fixed by 8a73b58267ebe373ae334e3f7c6ab42412a1933d